content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -148,7 +148,7 @@ These examples demonstrate the behavior of various dual-stack Service configurat
 1. This Service specification explicitly defines `IPv6` and `IPv4` in `.spec.ipFamilies` as well
    as defining `PreferDualStack` in `.spec.ipFamilyPolicy`. When Kubernetes assigns an IPv6 and
    IPv4 address in `.spec.clusterIPs`, `.spec.clusterIP` is set to the IPv6 address because that is
-   the first element in the `.spec.clusterIPs` array, overriding the default.
+   the first element in the `.spec.ipFamilies` array, overriding the default.
```

Comment:
Hmm. I think the original text is correct. Or you could try:
```suggestion
   IPv4 address in `.spec.clusterIPs`, Kubernetes sets `.spec.clusterIP` to the IPv6 address (because the
   IPv6 address comes first in the `.spec.clusterIPs` list).
```